contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -866,6 +867,17 @@ language client, and any other clients that use corresponding types
 Therefore, we ask that pointers always be used with optional fields that do not
 have a built-in `nil` value.
 
+## Nullable
+
+The `+nullable` comment tag allows the json `null` value to be a valid value
+for a field. The `null` value is serialized only when a field is a pointer
+in the Go definition, and does not have the `omitempty` json tag.
+
+For example, a map field marked with `+nullable` would accept either `foo: null` or `foo: {}`.
+
+Usage of `+nullable` is discouraged as it introduces several issues:
+- It is not compatible with json merge patching.
```

Comment:
(from the [merge patch RFC](https://datatracker.ietf.org/doc/html/rfc7386): "Null values in the merge patch are given special meaning to indicate the removal of existing values in the target.")